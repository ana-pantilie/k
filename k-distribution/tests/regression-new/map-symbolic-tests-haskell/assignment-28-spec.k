// Copyright (c) 2019 K Team. All Rights Reserved.
// generated by gen-tests.sh

requires "map-tests.k"

module ASSIGNMENT-28-SPEC
    imports MAP-TESTS

    claim <k> assignment ( ( foo ( X:MyId ) |-> BAL_TO:Int foo ( Y:MyId ) |-> BAL_FROM:Int MAP:Map ) [ foo ( Y ) <- BAL_FROM -Int 30 ] ) => assignmentResult ( ( foo ( X:MyId ) |-> BAL_TO:Int foo ( Y:MyId ) |-> BAL_FROM -Int 30 MAP:Map ) ) </k> requires       X   =/=K       Y

endmodule
