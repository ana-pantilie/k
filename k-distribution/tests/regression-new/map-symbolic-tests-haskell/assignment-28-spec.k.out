  #Not ( {
    X
  #Equals
    y
  } )
#And
  <k>
    assignmentResult ( ( X:MyId |-> 3
    y |-> 4 ) [ x <- 5 ] ) ~> .
  </k>
